transformations/gender_randomizer/transformation.py

```diff
@@ -0,0 +1,175 @@
+import random
+import re
+
+import coreferee
+import spacy
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Randomizes names in text for a 50/50 gender breakdown. Handles pronouns.
+"""
+nlp = spacy.load("en_core_web_sm", disable=["lemmatizer"])
+nlp.add_pipe("coreferee")
```

Comment:
You might want to use spacy like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/d39f0e5f57a79937296351996617ac03ee80fb74/transformations/close_homophones_swap/transformation.py#L55).

Reply:
Modified as given in example